dwave/plugins/torch/autoencoder/losses/kl_divergence.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2025 D-Wave
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Optional
+
+import torch
+from dimod import Sampler
+
+from dwave.plugins.torch.boltzmann_machine import GraphRestrictedBoltzmannMachine
+
+__all__ = ["pseudo_kl_divergence_loss"]
+
+
+def pseudo_kl_divergence_loss(
```

Comment:
We probably talked about this earlier, but is this only useful for the discrete autoencoder, or could it be more generic and thus make sense to put in `dwave.plugins.torch.losses` rather than in `dwave.pluginst.torch.autoencoder.losses`? @VolodyaCO @kevinchern

Reply:
sry totally missed this. I do not think it is generic enough to be in `dwave.plugins.torch.losses`